introduction.md

```diff
@@ -5,3 +5,11 @@ I like puppies, food, and puppies and more food weeee
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?